Api and bot init

bafaurazan opened the pull request.

bafaurazan committed:
adding subprocess availability

bafaurazan committed:
changing introduction

bafaurazan committed:
initializing django and discord_bot

documentation is available on /api/README.md file

bafaurazan committed:
changing dotenv form and api/README

- Using this dotenv method, environment variables are not affected
- Updating api/README, now it helps correctly

piasekj commented on api/README.md:
````diff
@@ -0,0 +1,30 @@
+- go to folder
+
+``` shell
+cd talk_bot/api/
````
```suggestion
cd api
```

piasekj commented on api/README.md:
````diff
@@ -0,0 +1,30 @@
+poetry run python open.py
+```
+
+## If you have strugle with python version, type following
````
```suggestion
## If you strugle with python version, type the following
```

bafaurazan committed:
Apply suggestions from code review

Co-authored-by: sand194 <108941271+sand194@users.noreply.github.com>

bafaurazan committed:
poetry update for django oops

piasekj commented on .gitignore:
```diff
@@ -1,160 +1,11 @@
+.envrc
 .venv
-env/
-venv/
```
bring him back pls, he useful boi

bafaurazan committed:
adding platform dependency

now only just for open.py file

bafaurazan committed:
adding in gitignore: env/ , venv/

piasekj commented on api/open.py:
```diff
@@ -1,16 +1,26 @@
+    return my_platform == "Windows"
+
+
+dependency = myPlatformDependency()
```
very nice

piasekj reviewed (approved):
![lord-farquaad-markiplier](https://github.com/bafaurazan/talk_bot/assets/108941271/672b54f2-e160-4f42-b61e-158abfded4ce)

piasekj commented on api/open.py:
```diff
@@ -1,16 +1,26 @@
 BASE_DIR = Path(__file__).resolve().parent
 
 
+def myPlatformDependency():
```
nitpick: maybe change the function name to checkPlatformDependency

bafaurazan committed:
changing function name

changing function name in file api/open.py , from myPlatformDependency to checkPlatformDependency
